fix(Migration guide): add "RDBMS requirements" section + improve indentation

ugaston opened the pull request:
Versions: 7.6, 7.7, 7.8, 7.9

Add a new section in the migration guide to make users aware of the need to have a DB version supported by the target Bonita version.

Additionally, improve the indentation for the sake of user-friendliness.

ugaston committed:
fix (Migration guide): add "RDBMS requirements" section + improve indentation

Versions: 7.6, 7.7, 7.8, 7.9

Add a new section in the migration guide to make users aware of the need to have a DB version supported by the target Bonita version.

Additionally, improve the indentation for the sake of user-friendliness.

tbouffard commented on md/migrate-from-an-earlier-version-of-bonita-bpm.md:
```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
+   * Please make sure to use the [appropriate JDBC driver](database-configuration.md#toc3)
+ :::
+
+**Community/Subscription edition:**
```
👍 good info

tbouffard commented on md/migrate-from-an-earlier-version-of-bonita-bpm.md:
```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
+
+**Community/Subscription edition:**
+The tool migrates your platform (_bonita_home_ folder and the database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating.
+* If you are running a Bonita Subscription Pack edition, you need a valid license for your target version.
```
⚠️ not sure `Pack` is the right wording
we may ask the product team if we are not sure about the right term to use

ugaston replied:
I didn't change the working here, I just took what was already in place.
Anyway, looking at Salesforce, our different subscriptions are called:
- Subscription Pack by Instances
- Subscription Pack Enterprise

So I guess this is right?
But I agree, it would be worth checking with Product team.

tbouffard replied:
Yes you are right I didn't see that this wording was already there
If wording must be changed, let's do it in another PR

tbouffard reviewed (commented):
Some minor changes may be implemented.
For info, you can also check #801 for an improvement about the same subject: Migration to 7.9+ when using Oracle db

educhastenier commented on md/migrate-from-an-earlier-version-of-bonita-bpm.md:
```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
+ :::
+
+**Community/Subscription edition:**
+The tool migrates your platform (_bonita_home_ folder and the database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating.
```
:question: Could we now remove reference to **bonita_home**, now that older versions are out of support?

```suggestion
The tool migrates your platform (Bonita database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating.
```

ugaston replied:
Well, they are indeed out of support, but precisely so, we may still have people coming from one of those versions....

educhastenier commented on md/migrate-from-an-earlier-version-of-bonita-bpm.md:
```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
+
+**Community/Subscription edition:**
+The tool migrates your platform (_bonita_home_ folder and the database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating.
+* If you are running a Bonita Subscription Pack edition, you need a valid license for your target version.
```
```suggestion
* If you are running a Bonita Subscription edition, you need a valid license for your target version.
```

ugaston replied:
Same answer as the one to Thomas' comment.

I'm fine to remove the 'Pack' if that is what Product recommends.

educhastenier commented on md/migrate-from-an-earlier-version-of-bonita-bpm.md:
```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
+* If you are upgrading to a new maintenance version and not changing the minor version number (for example, you are migrating from 6.3.0 to 6.3.1), your current license remains valid after migration.
 
 ::: info
 Starting from version 7.3 there is no more _bonita home_ folder. This means that, if your installation does not have any custom change, then you do not need to configure the bundle any further for an installation migrated in 7.3 or above.
```
If you answered yes to my first remark, we should also remove this § ...

ugaston committed:
fix(Migration guide): take Thomas' and Manu's comments into account

Fix links with anchors to Database configuration page.

Take Thomas' and Manu's comments into account.

ugaston committed:
fix(Migration guide): minor aspect fixes

Couple of tiny fixes.

tnguyen1 commented:
retest this please

tbouffard commented:
@educhastenier @ugaston what is the status of this old PR? can we merge it?

ugaston commented:
@tbouffard Definitely, as far as I'm concerned, if you are fine with it.

tbouffard commented:
Merging as all feedbacks have been integrated
